packages/trpc/server/routers/viewer/workflows/update.handler.ts

```diff
@@ -629,11 +621,9 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
   });
 
   if (addedSteps) {
-    const eventTypesToCreateReminders = activeOn.map((activeEventType) => {
```

Comment:
this resulted in having null and undefined values in `eventTypesToCreateReminders` which made `booking.findMany()` from [Line 644](https://github.com/calcom/cal.com/pull/12592/files#diff-e434eb7fdd98d2062613cf2c8f5a69bd969b932d5cd5b0ca329d434b118f6fa6R644) fail